instrumentation/logback-classic-1.2/build.gradle

```diff
@@ -0,0 +1,18 @@
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.logback-classic-1.2' }
+}
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation("ch.qos.logback:logback-core:1.2.6")
```

Comment:
Do we actually rely on anything in `logback-core`? I don't see mention of it in the instrumented classes.

Reply:
Indeed, we don't! I removed it.